### DISCUSSION TITLE
ABAQUS script not loads module when launched through command line

### DISCUSSION THREAD

**Discussion by ale-sepi** [haiiliin/abqpy#3879] [General]
I wrote a basic ABAQUS script in Python to extract the values of stresses of a node set. It works fine if I run it through ABAQUS CAE (File -> Run Script) but if I try to utilize the command line it spits out an error. I use the following command:

`abaqus cae noGUI=script.py`

Basically is supposed to be the same but without a graphical interface

```
from abaqus import *
from abaqusConstants import *
import os
import numpy as np
import displayGroupOdbToolset as dgo
from odbAccess import *
from driverUtils import *
import csv

#Open results file
odb = session.openOdb(name='C:\Users\...')
session.viewports['Viewport: 1'].setValues(displayedObject=odb)

#Filter Nodes
leaf = dgo.LeafFromNodeSets(nodeSets=('SET-10', ))
session.viewports['Viewport: 1'].odbDisplay.displayGroup.replace(leaf=leaf)

# Create Viewport      
session.viewports['Viewport: 1'].makeCurrent()
session.fieldReportOptions.setValues(reportFormat=COMMA_SEPARATED_VALUES)

#Extract Stress Values
session.writeFieldReport(fileName="data1.csv", append=ON,
    sortItem='Node Label', odb=odb, step=0, frame=1, outputPosition=NODAL,
    variable=(('S', INTEGRATION_POINT, ((COMPONENT, 'S11'),(COMPONENT, 'S22'), 
                                        (COMPONENT, 'S33'), (COMPONENT, 'S12'), 
                                        (INVARIANT, 'Mises'), )),))

print('Done')
```

The errors is: ImportError: No module named displayGroupOdbToolset

It seems that it cannot load this module that I utilize for filtering nodes. Am I missing something obvious?

**Comment by haiiliin** [comment]
It seems that this is the case you must open the GUI to run the script, but abqpy does not open GUI by default by using the noGUI option, you can add a new system environment variable ABAQUS_COMMAND_OPTIONS to "{'noGUI': False }", see here: https://docs.abqpy.com/en/latest/envvars.html

**Reply by ale-sepi** [reply]
Thank you for your response, I managed to set all accordingly to the documentation. Do you know some workaround for achieve the same result but without using "displayGroupOdbToolset"?

**Reply by haiiliin** [reply]
> Thank you for your response, I managed to set all accordingly to the documentation. Do you know some workaround for achieve the same result but without using "displayGroupOdbToolset"?

Sorry, I don't really know much about this.
